pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -291,8 +293,9 @@ message SplitAndScatterSpec {
   repeated RestoreEntryChunk chunks = 1 [(gogoproto.nullable) = false];
   repeated TableRekey table_rekeys = 2 [(gogoproto.nullable) = false];
   repeated TenantRekey tenant_rekeys = 3 [(gogoproto.nullable) = false];
+  optional jobs.jobspb.RestoreValidation validation = 5 [(gogoproto.nullable) = false];
```

Comment:
@dt note since the initial pr, I changed the `validation` field to not nullable.